spec.html

```diff
@@ -5188,6 +5194,9 @@ <h1>Abstract Equality Comparison</h1>
           1. If the mathematical value of _x_ is equal to the mathematical value of _y_, return *true*; otherwise return *false*.
         1. Return *false*.
       </emu-alg>
+      <emu-note>
+	      <p>Additional Equality Comparison steps related to [[IsHTMLDDA]] Internal Slot can be found in <emu-xref href="#sec-IsHTMLDDA-internal-slot-aec"></emu-xref>.</p>
```

Comment:
I would probably prefer for this note to be right in the algorithm steps, just before step 3 where the additional steps are to be inserted. It doesn't have to be an `<emu-note>` when it's inline in the algorithm steps, see [ApplyStringOrNumericBinaryOperator](https://tc39.es/ecma262/#sec-applystringornumericbinaryoperator) step 2 as an example.

@tc39/ecma262-editors Do you agree?

Reply:
SGTM

Reply:
I think a more apt reference would be similar to step 13 of [15.1.11](https://tc39.es/ecma262/#sec-globaldeclarationinstantiation) which is to add an additional step telling to replace it with annex b reference but that would need a slight change in [B.3.7.2](https://tc39.es/ecma262/#sec-IsHTMLDDA-internal-slot-aec) i.e. ` The following steps replace step 4 of the Abstract Equality Comparison algorithm: `

Reply:
@daemon1024 I think that would be a great improvement!